CHANGELOG.md

```diff
@@ -16,3 +16,4 @@ All notable changes to this project will be documented in this file.
 - @adamcohen - Expose `transaction_type` on transaction
 - @adamcohen - Expose `gateway_specific_response_fields` on transaction
 - @adamcohen - Expose `payment_method_type` on transaction
+- @patrickarnett - Modify `list_transactions` and `list_transactions_url` to take `count`, `order`, and `state` arguments in an optional hash.
```

Comment:
This should go in the `unreleased` section (right below line 5) and we'll pull it down into a release version when we cut a release.

Reply:
Oops!